portfolio/src/main/webapp/index.html

```diff
@@ -8,14 +8,31 @@
   </head>
   <body>
     <div id="content">
-      <img src="sheon_images/headshot.jpg" />
+      <img src="sheon_images/headshot.jpg" width="500" height="600"/>
       <h1>Sheon Mwapinza <a href="https://www.linkedin.com/in/sheon-mwapinza-963518196/">LinkedIn</a>, <a href="https://github.com/sheonkuda">GitHub</a>.</h1>
-      <p>My name is Sheon Mwapinza, I am from Zimbabwe! I am a sophomore at the University of Pennsylvanina 
-          and I am studying electrical engineering. I love sports, playing chess and I love playing with my baby brother.</p>
-      <p>My journey in computer science has been most fulfulling. Nothing compares to the feeling that mauls you after spending an entire night
-         awake working on the SPS portfolio and are finally done. Although painful, there is a silver lining to spending these nights awake,
+      <p>Quote of the day(see bottom of page):</p>
+      <button onclick="addRandomGreeting()">Quote</button>
```

Comment:
Maybe move this next to the greeting-container div, so that the quotes and the button show up next to each other? As a user, I would expect to see something happen right away when pressing the button, instead of needing to scroll the page to see something. Just a suggestion, up to you!

Reply:
yeah, I will fix this